src/ts_catalog/continuous_agg.h

```diff
@@ -135,12 +135,10 @@ ContinuousAggIsHierarchical(const ContinuousAgg *cagg)
 	return (cagg->data.parent_mat_hypertable_id != INVALID_HYPERTABLE_ID);
 }
 
-typedef enum ContinuousAggHypertableStatus
+typedef struct ContinuousAggHypertableStatus
```

Comment:
what is the expected `sizeof(struct ContinuousAggHypertableStatus)` here? It might be undefined behaviour how compiler will init this structure, the state of other 30 bits